static/app/actionCreators/pageFilters.tsx

```diff
@@ -406,7 +406,7 @@ async function persistPageFilters(filter: PinnedPageFilter | null, options?: Opt
  * Checks if the URL state has changed in synchronization from the local
  * storage state, and persists that check into the store.
  */
-async function updateDesyncedUrlState(router?: Router) {
+async function updateDesyncedUrlState(router?: Router, shouldForceProject?: boolean) {
```

Comment:
Wondering if we should make this parameter more clear as to what it does in relation to the `updateDesyncedUrlState` function?